src/System.Text.Json/roadmap/images/PlantUML-license.txt

```diff
@@ -0,0 +1,9 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+' Requires PlantUML to interpret: http://plantuml.com/
+' See license info under http://plantuml.com/faq for more details
```

Comment:
Why do we need/want any of this PlantUML stuff?  I don't see any other UML in corefx.

Reply:
I agree.  I don't think the UML/graphics add any real value to the readme.